assets/css/googlesitekit-trusted-login.css

```diff
@@ -0,0 +1,2 @@
+@import "../../../third-party/trustedlogin/client/src/assets/trustedlogin.css";
```

Comment:
This is not a requirement for integration, merely the result of a brief exploration into how we could modify the styles on the Trusted Login screen if so desired.